### PR TITLE
Update Inspector FAQ on in-app inspector "bubble"

### PR DESCRIPTION
**Pull request by stefaniabje**
Add two FAQs confirming that the in-app inspector "bubble" is off in prod.

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝Changed routes:


| [**/faqs/faq-inspector**](https://docs-ab2zmumv6.vercel.app/faqs/faq-inspector) |
|:-:|
|<a href="https://docs-ab2zmumv6.vercel.app/faqs/faq-inspector">Screenshot of /faqs/faq-inspector</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/obQzort39cOlIEYmBE2QUSwgOPM_.png">(view full size)</a> |







Commit 8b1c039244e82df7958a66ddb68b97e56b992fa6 (https://docs-ab2zmumv6.vercel.app).

File: pages/faqs/faq-inspector.mdx [hunk 1]
```diff
@@ -50,3 +50,15 @@ Reach out to support@avo.app or on Intercom, we would love to help!
 ### What kind of alerts can I set, if any?
 
 You can receive email alerts when new issues are detected. Head to the Issues dashboard on avo.app to configure email alerts.
+
+### When will the "bubble" from the In-App Inspector (aka Avo Debuggers) be visible in my app?
+
+The "bubble" of the [In-App Inspector (aka Avo Debuggers)](https://www.avo.app/docs/explore-tracking-plan/start-using-visual-debuggers) is visible when [the debugger is initialized](https://github.com/avohq/react-native-analytics-debugger) in a `dev` environment. Read more about the
```
Comment (logason) [inline pages/faqs/faq-inspector.mdx:56]: I think the answer to this question really depends on the platform you are using. We're linking to the standalone react-native debugger there, but afaik the visual debugger has also been embedded into all the Inspector SDKs.

The visual debugger is also made visible on web without any additional dependencies on web if you're running Avo Functions in dev mode.

@alex-tpom6oh could you summarize how you can get the In-App inspector up and running on each platform? Maybe those details can live in this faq, or we could link to more details somewhere else in the docs?

File: pages/faqs/faq-inspector.mdx [hunk 2]
```diff
@@ -50,3 +50,15 @@ Reach out to support@avo.app or on Intercom, we would love to help!
...
+
+The "bubble" of the [In-App Inspector (aka Avo Debuggers)](https://www.avo.app/docs/explore-tracking-plan/start-using-visual-debuggers) is visible when [the debugger is initialized](https://github.com/avohq/react-native-analytics-debugger) in a `dev` environment. Read more about the 
+
+### If I ship the Inspector SDK to prod, will the In-App Inspector bubble be visible in prod?
+
+Nope! 
+
+First of all, [setting up the Inspector SDK](https://www.avo.app/docs/implementation/setup-inspector-sdk) does not automatically activate the [In-App Inspector (aka Avo Debuggers)](https://www.avo.app/docs/explore-tracking-plan/start-using-visual-debuggers). Instead, the Inspector SDK activates the [Inspector Dashboard](https://www.avo.app/inspector) so you get ongoing data observability and can start figuring out what is wrong with tracking 😅
+
+Second of all: The "bubble" is only on in `dev`. (More in above answer).
```
Comment (logason) [inline pages/faqs/faq-inspector.mdx:64]: If you set the Inspector SDK to dev mode, the visual debugger will pop up, at least for the platforms that have them baked together.

I feel like we could simplify this answer down to the "second of all" part. We never pop up the bubble if you initialize the Inspector SDK with the env set to `prod`.

File: pages/faqs/faq-inspector.mdx (hunk 1 above)
Comment (alex-tpom6oh) [reply pages/faqs/faq-inspector.mdx:56]: There are 2 approaches in bundling the debugger currently.
The old one - when it is bundled with Avo functions and the new one - when it is bundled with Inspector.

The new one is currently available in Android and iOS. In this case the event path is `Avo Function | manual tracking -> Inspector -> Debugger`:
1. iOS - visible if Inspector SDK is included in the project and Inspector is initialized in dev or staging mode. (there is a method to show/hide the visual inspector, can use it in any environment). 
2. Android - visible if the `android-avo-inspector:dev:` gradle dependency is included in the project and Inspector is initialised in dev or staging mode. (there is a method to show/hide the visual inspector, can use it in any environment)
The `android-avo-inspector:prod:` gradle dependency does not have the visual inspector included.


The old one is on Web and React Native. In this case the event path is `Avo Function -> Debugger`:
1. Web - visible on without any additional dependencies on web if you're running Avo Functions in dev mode.
2. React Native - not shown automatically. You need to include `react-native-analytics-debugger` dependency, create instance of the debugger, pass it to initAvo and then call `AvoDebugger.showDebugger({mode: 'bar' // or 'bubble'})` as described here https://github.com/avohq/react-native-analytics-debugger

(Android and iOS also have this mechanism, but I think we should deprecate it and not mention in favour of the Inspector bundle)

File: pages/faqs/faq-inspector.mdx (hunk 2 above)
Comment (alex-tpom6oh) [reply pages/faqs/faq-inspector.mdx:64]: Needs a little tweak, now Android and iOS Inspector SDKs activate the bubble in all environments except prod.

File: pages/faqs/faq-inspector.mdx (hunk 1 above)
Comment (alex-tpom6oh) [reply pages/faqs/faq-inspector.mdx:56]: Not sure how deep into the details should we go here?

**Comment by alex-tpom6oh** [comment]
Maybe something like this would work, what do you think @logason ?

File: pages/faqs/faq-inspector.mdx [hunk 3]
```diff
@@ -50,3 +50,15 @@ Reach out to support@avo.app or on Intercom, we would love to help!
 ### What kind of alerts can I set, if any?
 
 You can receive email alerts when new issues are detected. Head to the Issues dashboard on avo.app to configure email alerts.
+
+### When will the "bubble" from the In-App Inspector (aka Avo Debuggers) be visible in my app?
+
+The "bubble" of the [In-App Inspector (aka Avo Debuggers)](https://www.avo.app/docs/explore-tracking-plan/start-using-visual-debuggers) is visible when Avo is used in a `dev` environment. The details differ between platforms as follows:
+
+On Android and iOS the bubble will be shown if you unclude the Inspector SDK and initialize it in the `dev` environment.
```
Comment (logason) [inline pages/faqs/faq-inspector.mdx:58]:
```suggestion
On Android and iOS the bubble will be shown if you include the Inspector SDK and initialize it with the environment parameter set to `dev`.
```

File: pages/faqs/faq-inspector.mdx [hunk 4]
```diff
@@ -50,3 +50,15 @@ Reach out to support@avo.app or on Intercom, we would love to help!
 ### What kind of alerts can I set, if any?
 
 You can receive email alerts when new issues are detected. Head to the Issues dashboard on avo.app to configure email alerts.
+
+### When will the "bubble" from the In-App Inspector (aka Avo Debuggers) be visible in my app?
+
+The "bubble" of the [In-App Inspector (aka Avo Debuggers)](https://www.avo.app/docs/explore-tracking-plan/start-using-visual-debuggers) is visible when Avo is used in a `dev` environment. The details differ between platforms as follows:
```
Comment (logason) [inline pages/faqs/faq-inspector.mdx:56]:
```suggestion
The "bubble" of the [In-App Inspector (aka Avo Debuggers)](https://www.avo.app/docs/explore-tracking-plan/start-using-visual-debuggers) is visible when Avo is initialized in `dev` mode. The details differ between platforms as follows:
```

File: pages/faqs/faq-inspector.mdx [hunk 5]
```diff
@@ -50,3 +50,15 @@ Reach out to support@avo.app or on Intercom, we would love to help!
...
+
+On Android and iOS the bubble will be shown if you unclude the Inspector SDK and initialize it in the `dev` environment.
+On Web the bubble will be shown if you are using Avo Functions in the `dev` environment.
```
Comment (logason) [inline pages/faqs/faq-inspector.mdx:59]:
```suggestion
On Web the bubble will be shown if you are using Avo Functions and the env parameter in `initAvo` is set to `dev`.
```

File: pages/faqs/faq-inspector.mdx [hunk 6]
```diff
@@ -50,3 +50,15 @@ Reach out to support@avo.app or on Intercom, we would love to help!
...
+On react Native the bubble is not shown automatically. You would have to trigger it manually.
```
Comment (logason) [inline pages/faqs/faq-inspector.mdx:60]:
```suggestion
On React Native the bubble is not shown automatically. You will have to trigger it manually.
```

Comment (logason) [reply pages/faqs/faq-inspector.mdx:60]: Maybe add a "Learn more about how to control the bubble in React Native" link here?

File: pages/faqs/faq-inspector.mdx [hunk 7]
```diff
@@ -50,3 +50,15 @@ Reach out to support@avo.app or on Intercom, we would love to help!
...
+
+### If I ship the Inspector SDK to prod, will the In-App Inspector bubble be visible in prod?
+
+Nope! The "bubble" is only on in `dev`. (More in above answer).
```
Comment (logason) [inline pages/faqs/faq-inspector.mdx:64]:
```suggestion
Nope! The "bubble" is only on when env is set to `dev`. (More in above answer).
```

**Review by logason — COMMENTED** [review]
@alex-tpom6oh I like this!

Added few minor suggestions, clarifying that this is controlled by the env parameter when initializing Avo (but not automatically inferred from the environment).
